docs/content/en/docs-dev/user-guide/managing-piped/configuring-event-watcher.md

```diff
@@ -27,6 +27,8 @@ spec:
 ```
 
 ### [optional] Specify Eventwatcher files Piped will use
+>NOTE: This way is valid only for defining events using `.pipe/`.
```

Comment:
should we add a link to the docs to this `.pipe/` pattern. I feel the docs are a bit complicated here.